pkg/parquet/reader.go

```diff
@@ -37,8 +37,8 @@ type (
 type Reader interface {
 	ReadDate(ctx context.Context, datetime time.Time, target interface{}) error
 	ReadDateAsync(ctx context.Context, datetime time.Time, target interface{}, callback ResultCallback) error
-	ReadFileIntoTarget(ctx context.Context, file string, target interface{}) error
-	ReadFile(ctx context.Context, file string) (ReadResults, error)
+	ReadFileIntoTarget(ctx context.Context, file string, target interface{}, batchSize *int, offset int64) error
+	ReadFile(ctx context.Context, file string, batchSize *int, offset int64) (ReadResults, error)
```

Comment:
i think if we want to make it the way it is done in the go sdk, then we'd not make it pointers but rather check for a -1 value => infinite